### PR TITLE
Feature/dark mode

### PR DESCRIPTION
**Pull request by augustskare**
I hope I didn't change to much of the design when moving all colors over to variables. But now, if the users prefers dark mode, that's what they get! 💃 

ezgif com-video-to-gif

### CODE REVIEW

File: src/components/base.css [hunk 1]
```diff
@@ -1,25 +1,45 @@
...
+  --purple-dark: #9a98d4;
+  --vaiant1: #fff;
+  --vaiant2: #f4f4f4;
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --background: #272727;
+    --color: #fff;
+    --vaiant1: #3a3a3a;
```
Comment (ezekielaquino) [inline src/components/base.css]: Also just gonna be nitpicky here, shall we rename "vaiant" to something else, more descriptive perhaps?

Comment (augustskare) [reply src/components/base.css]: I was trying to think of something more descriptive but could not find out what it should be 🙈 

`--vaiant2` is used for the gray-ish inputs and `--variant` is the background of heading input and the box around the status item. Do you have any suggestions for better names? 👼

**Comment by augustskare** [comment]
I'll fix the conflict with the image :)

File: src/components/base.css (hunk 1 above)
Comment (ezekielaquino) [reply src/components/base.css]: ah! sorry, maybe just `variant`? :D  then it makes sense

Comment (augustskare) [reply src/components/base.css]: agree, I'll make the changes :)

**Pull request merged**